src/index.js

```diff
@@ -226,6 +226,46 @@ export default async function microbundle(options) {
 	);
 }
 
+function createBabelConfig(options) {
+	const partialConfig = loadPartialConfig();
+	const hasBabelConfigFile = partialConfig.hasFilesystemConfig();
+
+	// Allow project to configure babel with local config file
+	if (hasBabelConfigFile) return {};
+
+	// No babelrc exists, use default config
+	// TODO move this into a separate babel-preset-microbundle package
+	return {
+		babelrc: false,
+		extensions: [...DEFAULT_EXTENSIONS, '.ts', '.tsx'],
```

Comment:
atm `rollup-plugin-typescript2` is responsible for transpiling typescript and we should keep it - especially in this PR. We can decide to transpile TS with babel later on, but it has some caveats so it needs extra consideration.

Reply:
Oh, ok - makes sense. I was not aware of the fact that the transpilation has been splitted into 2 steps.

I see that u have updated the before/after table. Super cool 👏 Did you also update the gists?

Reply:
Yep, it's updated :) The following files have changed:
- basic-lib-ts.js 
- basic-lib-tsx.js 
- class-decorators-ts.js
- esnext-ts.js (Removed because test fails)

Reply:
Good point! Babel would not work on typescript files if the user also has a babel config file.

This entire logic may change when we add rollup-plugin-babel custom loader, but let's add it for now.